cost/google/recommender/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.2.1
+
+- Fetching only ACTIVE recommendations from Google Cloud Recommender API. Functionality unchanged.
```

Comment:
Should be reworded to make it clear that this happens during policy execution. We also don't want to say "Functionality unchanged" because it did, in fact, change.

- Policy now only fetches active recommendations from Google Cloud Recommender API.